trace_state/HTTP_HEADER_FORMAT.md

```diff
@@ -33,20 +33,20 @@ All spaces MUST be trimmed from the beginning and the end of the value. Value en
 
 Since name value pairs are well-known for vendor's library - it is not recommended to use properties for `Trace-Parent` key value pairs. Vendor-specific library would typically know how to parse and interpret the value for every name.
 
-Properties are expected to be in a format of keys & key-value pairs `;` delimited list `;k1=v1;k2;k3=v3`. All unknown property names and name-value pairs MUST be preserved. 
+Properties are expected to be in a format of keys & key-value pairs `;` delimited list `k1=v1;k2;k3=v3`. All unknown property names and name-value pairs MUST be preserved. 
 
 # Examples of HTTP headers
 
 Single header: 
 
 ```
-Trace-State: parent_application_id = 123
+Trace-State: parent_application_id=123
```

Comment:
spaces were intentional to demo that you CAN do spaces here. Do you think we should have two examples - one specifically to demonstrate spaces trimming?